datalake/src/main/java/com/sequenceiq/datalake/flow/datalake/upgrade/event/DatalakeUpgradeStartEvent.java

```diff
@@ -32,15 +32,18 @@ public boolean isReplaceVms() {
         return replaceVms;
     }
 
-    @Override
-    public String selector() {
```

Comment:
Thinking point: won't it cause an issue?

Reply:
as the class is initialized with it's constructor using this: https://github.com/hortonworks/cloudbreak/blob/master/datalake/src/main/java/com/sequenceiq/datalake/flow/datalake/upgrade/DatalakeUpgradeEvent.java#L7
it would result in the same selector. either the constructor or this should be removed as it was misleading.